rgw/v2/tests/s3_swift/multisite_configs/test_multisite_bucket_mirror_sync_policy.yaml

```diff
@@ -0,0 +1,17 @@
+# Polarian TC CEPH-83575138
+config:
+  user_count: 1
+  bucket_count: 1
```

Comment:
Could you please increase the bucket count

Reply:
Increased to 2